ml_server/main.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pathlib import Path
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from typing import List
+from functools import lru_cache
+
+from fastapi import Depends
+from gliner import GLiNER
+
+from tenant_dependency import get_tenant_info
```

Comment:
You did not specify this dependency in project requirements

Reply:
I've added it to Readme. And as I understand other services do the same. Because you either use python_base image (that already have those installed) or you install it locally then you go by Readme.